packages/domain-lib/src/aggregate.ts

```diff
@@ -62,7 +63,6 @@ export class Aggregate {
 		TIME_ZONE: string,
 		TIMEOUT_MS_LOCK_ACQUIRED: number,
 		MIN_DURATION_MS_TASK: number,
-		timeoutMsHttpClient: number
```

Comment:
How is timeout being handled after this removal? Even if fetch has a default value it might be way too long or too short for the purpose of the code, it's probably worth it to be able to keep control of it on the BC's service side.

Reply:
Fetch does not have native timeout support but there is a way to implement configurable timeouts that I found based on research